src/loader/x86_64/elf/mod.rs

```diff
@@ -130,7 +130,8 @@ impl Elf {
 impl KernelLoader for Elf {
     /// Loads a kernel from a vmlinux elf image into guest memory.
     ///
-    /// The kernel is loaded into guest memory at offset `phdr.p_paddr` specified by the elf image.
+    /// By default, the kernel is loaded into guest memory at offset `phdr.p_paddr` specified by the elf image.
+    /// When used, `kernel_offset` specifies a fixed offset from `phdr.p_paddr` at which to load the kernel.
```

Comment:
I think it's useful to explicitly document that whenever `kernel_offset` is present, the `pvh_entry_addr` field of the result will not be populated.

Reply:
Agree. Added that line to the comment.